build.sbt

```diff
@@ -13,4 +14,24 @@ lazy val microservice = Project(appName, file("."))
   .settings(publishingSettings: _*)
   .settings(resolvers += Resolver.jcenterRepo)
   .settings(PlayKeys.playDefaultPort := 9481)
+  .settings(
+    name := appName,
+    RoutesKeys.routesImport += "models._",
+    TwirlKeys.templateImports ++= Seq(
+      "play.twirl.api.HtmlFormat",
+      "play.twirl.api.HtmlFormat._",
+      "uk.gov.hmrc.play.views.html.helpers._",
+      "uk.gov.hmrc.play.views.html.layouts._",
+      "controllers.routes._"
+    ),
+    retrieveManaged := true,
+    evictionWarningOptions in update :=
+      EvictionWarningOptions.default.withWarnScalaVersionEviction(false),
+//    Concat.groups := Seq(
```

Comment:
commented out bits?

Reply:
Yea these lines were causing complications following the library upgrades and we'll potentially be coming back to them down the line